etc/browser-client.api.md

```diff
@@ -46,6 +46,7 @@ export interface ClientOptions {
     language?: string;
     loginUrl?: string;
     microphone?: Microphone;
+    projectId: string;

```

Comment:
Is projectId a mandatory parameter? If not, should it be marked as `projectId?`

Reply:
Its optional. Fixed.